claat/proto-renderer/testing-utils/proto_constructors.go

```diff
@@ -39,3 +39,57 @@ func NewInlineCode(txt string) *tutorial.InlineCode {
 		Code: txt,
 	}
 }
+
```

Comment:
I'm not clear on why we need these - can't the code calling these methods just call the constructors directly, like these methods are doing?

Reply:
There's no constructors in Go since there are no classes. Only way to instantiate elements is by explicit definition. These are such constructors. I got this from Cassie's PR ... TBA

Reply:
this also makes tests easier to write and read, as per #292: 
https://github.com/googlecodelabs/tools/blob/e57932077015ed9a5f4ecda80e2eeec99d7793dc/claat/proto-renderer/html/templates-tests/textblock_test.go#L24